CONTRIBUTING.md

```diff
@@ -276,6 +270,36 @@ Please help in ensuring all relevant issues are closed and that any subsequent i
 - If an issue in Core will affect a component in PF-React, this issue should link to the main PF-React issue.
 - The CSS Developers and UX Designers should be tagged to review their respective PF-React issue.
 
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**Build Failures:**
+- Ensure you're using Node.js version 22 or higher: `node --version`
+- Clear node_modules and reinstall: `rm -rf node_modules && yarn install`
+- Clear yarn cache: `yarn cache clean`
+
+**Test Failures:**
+- Run tests in watch mode for debugging: `yarn test --watch`
+- Update snapshots if components have intentionally changed: `yarn test -u`
+
+**Linting Errors:**
+- Auto-fix linting issues: `yarn lint --fix`
+- Format code with Prettier: `yarn prettier`
+
+**Development Server Issues:**
+- Clear cache and restart: `rm -rf node_modules/.cache && yarn start`
```

Comment:
May also want to add a note about clearing the `rm -rf packages/react-docs/.cache` as well, or update this line to point at react-docs instead of node_modules.